src/admin/index.js

```diff
@@ -428,38 +431,43 @@ module.exports = ({
 
   /**
    * @param {string} groupId
-   * @param {string[]]} topics
+   * @param {string[]} topics
    * @param {boolean} [resolveOffsets=false]
    * @return {Promise}
    */
-  const fetchConsumerGroupOffsets = async ({ groupId, topics, resolveOffsets = false }) => {
+  const fetchConsumerGroupOffsets = async ({ groupId, topics = [], resolveOffsets = false }) => {
     if (!groupId) {
       throw new KafkaJSNonRetriableError(`Invalid groupId ${groupId}`)
     }
 
+    if (!Array.isArray(topics)) {
+      throw new KafkaJSNonRetriableError(`Expected topics array, got ${topics}`)
+    }
+
     const coordinator = await cluster.findGroupCoordinator({ groupId })
 
     let consumerOffsets
     if (topics.length) {
-      const topicsToFetch = Promise.all(
+      const topicsToFetch = await Promise.all(
```

Comment:
Huh, how could this test have been passing before? 🤔  I suspect owls in the bog, as we say in Swedish...

Reply:
Yeah that was a change from the draft PR that had no tests so ofc it didn't cause any issues 😄